lustre/datadog_checks/lustre/check.py

```diff
@@ -0,0 +1,553 @@
+            try:
+                stat_dict: Dict[str, Union[int, str]] = {'count': int(parts[1])}
+                stat_types = ('min', 'max', 'sum', 'sumsq')
+                if len(parts) < 3:
+                    self.log.debug('Unexpected format for stat "%s"', line)
+                    continue
+                stat_dict['unit'] = parts[3].strip('[]')
+                for i, value in enumerate(parts[4:]):
+                    stat_dict[stat_types[i]] = int(value)
+                if len(parts) > 8:
+                    self.log.debug('Unexpected format for stat "%s"', line)
+            except ValueError:
+                self.log.debug('Could not parse stat value for "%s"', line)
+                continue
+            stats[stat_name] = stat_dict
+        return stats
+
+    def submit_device_health(self, devices: List[Dict[str, Any]]) -> None:
+        '''
+        Submit device health metrics based on device status from lctl dl command.
+        '''
+        try:
+            for device in devices:
+                device_status = 1 if device['status'] == 'UP' else 0
+                tags = [
+                    f'device_type:{device.get("type", "unknown")}',
+                    f'device_name:{device.get("name", "unknown")}',
+                    f'device_uuid:{device.get("uuid", "unknown")}',
+                ]
+                tags += self.tags
+
+                self._submit('device.health', device_status, 'gauge', tags=tags)
+                self._submit('device.refcount', device['refcount'], 'count', tags=tags)
+        except Exception as e:
+            self.log.error('Failed to submit device health metrics: %s', e)
+
+    def submit_changelogs(self, lines: int) -> None:
+        '''
+        Get changelogs from the command line.
+
+        For more information, see: https://doc.lustre.org/lustre_manual.xhtml#lustre_changelogs
+        '''
+        for target in self.changelog_targets:
+            changelog = self._get_changelog(target, lines)
+            changelog_lines = changelog.splitlines()
+            for line in changelog_lines:
+                if not line.strip():
+                    continue
+                parts = line.split()
+                try:
+                    date_time = parts[3] + ' ' + parts[2]
+                    # The time has nanoseconds, so we need to truncate the last three digits
+                    timestamp = (
+                        datetime.strptime(date_time[:-3], '%Y.%m.%d %H:%M:%S.%f')
+                        .replace(tzinfo=timezone.utc)
+                        .timestamp()
+                    )
+                    data = {
+                        'operation_type': parts[1],
+                        'timestamp': timestamp,
+                        'flags': parts[4],
+                        'message': ' '.join(parts[5:]),
+                    }
+                except IndexError:
+                    self.log.warning('Skipping changelog due to unexpected format: %s', line)
```

Comment:
```suggestion
                    self.log.debug('Skipping changelog due to unexpected format: %s', line)
```
I think this may be too high of a severity